packages/salesforcedx-apex-debugger/src/messages/i18n.ts

```diff
@@ -32,5 +32,9 @@ export const messages = {
   hotswap_warn_text:
     "You can't modify Apex classes or triggers during an Apex Debugger session. Save your changes after you're done debugging.",
   created_exception_breakpoint_text: 'Created exception breakpoint for %s.',
-  removed_exception_breakpoint_text: 'Removed exception breakpoint for %s.'
+  removed_exception_breakpoint_text: 'Removed exception breakpoint for %s.',
+  idle_warn_text:
+    'You have been idle for %s minutes. Your debugger session will be terminated in %s minutes if it continues to be idle.',
+  idle_terminated_text:
+    'Your debugger session is being terminated because it has been idle for %s minutes.'
```

Comment:
@ruthemmanuelle , see screenshot above for these messages. Is this enough to suggest that the user needs to do something with the debugger to not have their session terminated?

Reply:
Would something like this work for idle_warn_text?
'You have been idle for %s minutes. To prevent your debugger session from being terminated, run or step through code, or inspect your variables, in the next %s minutes.'